redaxo/src/addons/install/lib/api_package_update.php

```diff
@@ -112,6 +112,10 @@ public function doAction()
 
         $this->addon->setProperty('version', $this->file['version']);
         rex_install_packages::updatedPackage($this->addonkey, $this->fileId);
+
```

Comment:
kommentar an beiden stellen: `// re-generate cache to make sure new classes immediately are available`